### PR TITLE
Add static-gatsby example

### PR DESCRIPTION
**Pull request by sergiodxa**
Except the README, Dockerfile, .dockerignore and now.json the rest of the files are the default from `gatsby new` command

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -14,6 +14,7 @@ These deployments:
 
 | Demo  | Description | Size |
 |:------|:------------|:---- |
+| [static-gatsby](https://static-gatsby.now.sh) | A  static site built with [Gatsby](https://www.gatsbyjs.org/) | 3.1M |
```
Comment (sergiodxa) [inline README.md]: 3.1M is correct, there is a super big `.map` file

File: static-gatsby/gatsby-browser.js [hunk 2]
```diff
@@ -0,0 +1,7 @@
+/**
+ * Implement Gatsby's Browser APIs in this file.
+ *
+ * See: https://www.gatsbyjs.org/docs/browser-apis/
+ */
+
+ // You can delete this file if you're not using it
```
Comment (TooTallNate) [inline static-gatsby/gatsby-browser.js]: Let's delete this?

File: static-gatsby/gatsby-node.js [hunk 3]
```diff
@@ -0,0 +1,7 @@
+/**
+ * Implement Gatsby's Node APIs in this file.
+ *
+ * See: https://www.gatsbyjs.org/docs/node-apis/
+ */
+
+ // You can delete this file if you're not using it
```
Comment (TooTallNate) [inline static-gatsby/gatsby-node.js]: Same.

File: static-gatsby/gatsby-ssr.js [hunk 4]
```diff
@@ -0,0 +1,7 @@
+/**
+ * Implement Gatsby's SSR (Server Side Rendering) APIs in this file.
+ *
+ * See: https://www.gatsbyjs.org/docs/ssr-apis/
+ */
+
+ // You can delete this file if you're not using it
```
Comment (TooTallNate) [inline static-gatsby/gatsby-ssr.js]: Same.

File: static-gatsby/package-lock.json [hunk 5]
```diff
@@ -0,0 +1,12150 @@
+{
```
Comment (TooTallNate) [inline static-gatsby/package-lock.json]: There's appears to be both a `yarn.lock` and a `package-lock.json` file. Let's stick with just one, probably yarn.

File: static-gatsby/.dockerignore [hunk 6]
```diff
@@ -0,0 +1,8 @@
+*
+!src
+!gastsby-browser.js
```
Comment (TooTallNate) [inline static-gatsby/.dockerignore]: These can be removed too

**Pull request merged**